Makefile

```
@@ -57,13 +57,18 @@ push_translations: ## push source strings to Transifex for translation
 
 pull_translations:  ## pull translations from Transifex
 	git clean -fdX conf/locale
+ifeq ($(OPENEDX_ATLAS_PULL),)
 	i18n_tool transifex pull
 	i18n_tool extract
 	i18n_tool dummy
 	i18n_tool generate --verbose 1
 	git clean -fdX conf/locale/rtl
 	git clean -fdX conf/locale/eo
 	i18n_tool validate --verbose
+else
```

@OmarIthawi many of `i18n_tool` command are introducing segmented files. This is why I made them run only for non-atlas

---

Thanks @shadinaif, please add a TODO here and link to an `i18n-tools` repo issue.

Fixing this would be out of scope of the FC-0012, but it's worth noting because some engineers rely on this functionality for testing translation without relying on Transifex.

```suggestion
else
	# TODO: Generate dummy locales: Esperanto (eo) and RTL once `i18n_tools dummy` supports the `--no-segment` option: https://github.com/openedx/i18n-tools/issues/???
```